Week 3/Solution.java

```diff
@@ -0,0 +1,64 @@
+import java.io.*;
```

Comment:
Good job with this question - making use of the 2Sum algorithm is definitely a great way to go